test/test-browser-global.js

```diff
@@ -0,0 +1,14 @@
+"use strict";
+
+const {assert} = require("chai");
+
+const {setupTestDOMWindow} = require("./setup");
+
+describe("browser-polyfill", () => {
+  it("automatically wrapps chrome into a browser object", () => {
```

Comment:
*wraps

Also, probably something like "wraps the global `chrome` namespace with a global `browser` namespace"